### PR TITLE
fix reading calcul

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

**Comment by fuentesloic** [comment]
@vinyll I'm trying to change the reading time calcul from front to back.
The front calcul was taking the content lenght but:
- It's a calcul for the back I think
- The content is polluted by the image lenght who count as a lot of word and exagerate the reading calcul (around 245 minutes reading by image)

I'm pretty block to do that :s

File: blog/models.py [hunk 1]
```diff
@@ -33,14 +33,14 @@ def clean_html(html, *args, **kwargs):
 
 
 class Article(db.Document):
+    creation_date = db.DateTimeField(default=datetime.datetime.utcnow)
+    publication_date = db.DateTimeField()
```
Comment (vinyll) [inline blog/models.py]: Why moving this?

File: templates/blog/edit.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
         <a href={{ url_for_user(article.author) }}>{{ article.author.username }}</a>
         <p>{{ article.author.about|htmlnewline }}</p>
         <time datetime={{ article.creation_date }} pubdate>{{ article.creation_date.strftime('%d %B, %Y') }}</time>
-        <p class=reading-time><span>{{ (article.content|length / 6 / 250)|int }}</span> min read</p>
+        <p class=reading-time><span>{{ (article.content|striptags|length) / 6 / 250)|int }}</span> min read</p>
```
Comment (vinyll) [inline templates/blog/edit.html:24]: Does it work?

File: blog/models.py (hunk 1 above)
Comment (fuentesloic) [reply blog/models.py]: I would put all the "time" together

File: templates/blog/edit.html (hunk 2 above)
Comment (fuentesloic) [reply templates/blog/edit.html:24]: yes!! I'm trying to find a way and have a 1 min minimum... because for short article we have 0 ... it's weird

**Comment by fuentesloic** [comment]
@vinyll why test not succeed ??? I re put stuff at the good place ...

**Comment by vinyll** [comment]
Hey, I've [created a new PR](https://github.com/neomad-team/neomad.org/pull/298) inspired by yours.
That's the appropriate way we should have gone to earlier but was too distracted to get this done :/

**Pull request closed without merging**